rs/bitcoin/kyt/src/dashboard.rs

```diff
@@ -0,0 +1,107 @@
+use crate::{state, BtcNetwork};
```

Comment:
nit: style question, why not having this file as `rs/bitcoin/kyt/src/dashboard/mod.rs` so that it sits next to its unit tests `rs/bitcoin/kyt/src/dashboard/tests.rs`?

Reply:
I think the mono repo accepts either way. I kind of prefer a module sits at a level above its sub modules.

Reply:
>I think the mono repo accepts either way. 
yes totally, was asking out of curiosity (because personally I would go the other way, but that's totally subjective)

> I kind of prefer a module sits at a level above its sub modules.
I see, this is also a very good reason.